### ISSUE TITLE
Define the receive classes

### ISSUE THREAD

**Issue by tatsumibruno** [tecsinapse/tecsinapse-data-sync#12]
We have implemented the listener that identify the changed entities, now we need the other side, that will receive the data(as json) and merge it in the database.

cc @filipenevola @leonardoterrao

**Comment by leonardoterrao** [comment]
Let's try to follow some of the premises that we talk about @filipenevola  and @Janario in the first few conversations.

The other side should be generic. From the JSON that receives it knows how to identify which type and then follow a path according to the type.

I don't know exactly how, but let's talk and we can define an approach.

Do you agree @tatsumibruno?

**Comment by tatsumibruno** [comment]
Yes @leonardoterrao! Lets discuss about that. Talking with Henrique Zerbato, he suggested that we use an approach similar to Graphql, where the client tell what he want. We can talk with him about this idea, sounds good for me, what you think?

cc @filipenevola

**Comment by tatsumibruno** [comment]
Hi guys! I had an idea about this issue. Thinking about the approach suggested by Henrique, we could use a kind of publish/subscribe from meteor. I thought in this steps:
